### ISSUE TITLE
Need a script that restores database

### ISSUE THREAD

**Issue by aaronhelton** [dag-hammarskjold-library/dlx-rest#505]
A standalone script for refreshing data from production backup is easier to document. We should arguably have some of this built into DLX, but an initial script can get us started.

**Comment by viola-v** [comment]
@aaronhelton  Could you please check if this can be closed?

**Comment by aaronhelton** [comment]
I don't think this is necessary anymore. The current method for restoring a database would use the MongoDB Atlas console. I'm closing it for now, but we can reopen later if we determine we need it.
